routes/v1.js

```diff
@@ -256,11 +256,13 @@ api.declare({
     "",
     "**Task specific routing-keys**, using the `task.routes` property you may",
     "define task specific routing-keys. If a task has a task specific ",
-    "routing-key: `<route>`, then the poster will be required to posses the",
-    "scope `queue:route:<route>`. And when the an AMQP message about the task",
```

Comment:
That was intentional - it was moved into the payload definition.

The reason for introducing this change was that some scope requirements were defined in the API documentation (e.g. `queue:route:<route>` as above) yet some scope requirements were defined in the schema definition (e.g. `queue:task-priority:high`). For all the properties in the TaskDefinition, several fields have scope requirements on the poster of the task definition - yet only `queue:route:<route>` scope requirement was documented in the API definition, the others were all in the schema definition.

In this change I moved all this scope requirement information into the documentation of the fields they relate to, and put an **Important** heading in the API documentation to say _read the schema docs_ to find out extra scopes you might require to post a task. I also moved the information into a structured form, so the scope requirements were not mentioned in free text form arbitrarily in the middle of a definition - instead in a section called "Task submitter required scopes" at the end of the property definition.